### PR TITLE
Add PullRequests supports & Alter some team relationship config

### PR DESCRIPTION
**Pull request by Andrewmatilde**
PullRequests supports : just like issue supports, have relationship with label&user...
Alter some team relationship config : remove sig/migrate from TP-Storage Cloud-Storage team.

### CODE REVIEW

**Comment by Andrewmatilde** [comment]
PTAL @mahjonp

File: infrastructure/main.go [hunk 1]
```diff
@@ -2,15 +2,17 @@ package main
 
 import (
 	"fmt"
+	"github.com/PingCAP-QE/dashboard/infrastructure/codecov"
```
Comment (mahjonp) [inline infrastructure/main.go:5]: sort imports, please.

Comment (Andrewmatilde) [reply infrastructure/main.go:5]: ok

**Review by mahjonp — APPROVED** [review]

**Review by zhouqiang-cl — APPROVED** [review]
LGTM

**Pull request merged**